Assignment-2/src/main/java/application/MainClass.java

```diff
@@ -0,0 +1,73 @@
+package application;
+
+import java.util.List;
+import java.util.Scanner;
+
+import models.Display;
+import models.UserClass;
+import services.ServicesHelper;
+import services.SortServiceHelper;
+
+/**
+ * Main class.
+ */
+public final class MainClass {
+  private MainClass() {
+
+  }
+
+  /**
+   * Main function.
+   * @param args represents args
+   */
+  public static void main(String[] args) {
+    final String filePath = "users.json";
+
+    final List<UserClass> users = ServicesHelper.loadUsers(filePath);
+
+    final List<Integer> rolls = ServicesHelper.loadRollNum(users);
+
+    final Display content = new Display();
+
+    boolean flag = true;
+    final Scanner sc = new Scanner(System.in);
+    while (flag) {
+      int choice;
+      System.out.print(content.getDisplayDetails());
+      choice = sc.nextInt();
+
+      switch (choice) {
+        case 1:
+          // Add User
+          sc.nextLine();
+          ServicesHelper.addUser(users, rolls, sc);
+          break;
+        case 2:
+          // Display User
+          int sortChoice;
+          System.out.print(content.getDisplayUserBy());
+          sortChoice = sc.nextInt();
+          SortServiceHelper.sort(sortChoice, users);
+          ServicesHelper.displayUsers(users, content);
+          break;
```

Comment:
a function should ideally do one thing. 
Revisit this function to see if this can be broken up into concise functions.

Reply:
ok will change it